src/radical/pilot/agent/executing/shell.py

```diff
@@ -124,7 +124,9 @@ def initialize_child(self):
                                      # are not yet known
 
         # get some threads going -- those will do all the work.
-        import saga.utils.pty_shell as sups
+        try:    import         saga.utils.pty_shell as sups
+        except: import radical.saga.utils.pty_shell as sups
```

Comment:
Since we are preparing for sage renaming shouldn't it be:
```
        try: import radical.saga.utils.pty_shell as sups
        except: import saga.utils.pty_shell as sups
```
This way if radical.saga is installed everything is great otherwise grab saga?

Reply:
Ack, will switch.